Heap/Heap.swift

```diff
@@ -4,58 +4,55 @@
 //
 
 public struct Heap<T> {
+  
   /** The array that stores the heap's nodes. */
-  var elements = [T]()
+  private(set) var nodes = [T]()
```

Comment:
Let's declare this as simply `private` instead. The internal storage of the heap shouldn't be something people need to inspect.